### PR TITLE
2.9 download repository policy

### PR DESCRIPTION
**Pull request by vpereira**
Backport from PR #7165

### CODE REVIEW

**Comment by vpereira** [comment]
please check @hennevogel

**Comment by hennevogel** [comment]
@vpereira are you going to address the test failure or do you want @Ana06 to take over?

**Comment by vpereira** [comment]
> @vpereira are you going to address the test failure or do you want @Ana06 to take over?

Yes, I pushed already some fixes. It should be fixed by now

**Comment by vpereira** [comment]
@hennevogel even after rebased on upstream/2.9 haikiri still red :(

**Comment by Ana06** [comment]
@vpereira hakiri is complaining about the Rails security issue, which has to be solved by updating the gem. Check https://github.com/openSUSE/open-build-service/pull/7181 In other words, it is unrelated to this PR. Circle-ci should be green though

**Comment by vpereira** [comment]
> @vpereira hakiri is complaining about the Rails security issue, which has to be solved by updating the gem. Check #7181 In other words, it is unrelated to this PR. Circle-ci should be green though

hey @Ana06, [AFAIK we do not have CircleCi for branch 2.9](https://github.com/openSUSE/open-build-service/issues/7135), just Travis. The gem update in the branch 2.9 should follow the old strategy, as described here https://github.com/openSUSE/open-build-service/wiki/GEM-bundle. The security issues were published last Wednesday, and we should give a closer look maybe on that https://hakiri.io/github/openSUSE/open-build-service/2.9/8ba2655f3666517e8e347dffaa1d8a40fec7b1f9/warnings?name=File+Access, but as you said it's not related with this PR.

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ WORKDIR /obs/src/api
 # Installing the gem directly spares us from having to rpm package two different thor versions.
 RUN sudo gem.ruby2.5 install thor:0.19 foreman
 # Ensure there is a foreman command without ruby suffix
-RUN sudo ln -s /usr/bin/foreman.ruby2.5 /usr/bin/foreman
+#RUN sudo ln -s /usr/bin/foreman.ruby2.5 /usr/bin/foreman
```
Comment (bgeuken) [inline Dockerfile]: Was this change intended? If it was, please add a short explanation to the commit message. Otherwise just drop it:-)
